bentoml/yatai/deployment/aws_lambda/lambda_app.py

```diff
@@ -62,18 +66,36 @@ def api_func(event, context):  # pylint: disable=unused-argument
     Application Load Balancer, in which case the parameter `event` must be type `dict`
     containing the HTTP request headers and body.
 
-    see: https://docs.aws.amazon.com/lambda/latest/dg/services-alb.html
+    You can find an example of which
+    variables you can expect from the `event` object on the AWS Docs, here
+    https://docs.aws.amazon.com/lambda/latest/dg/services-alb.html
     """
 
     if type(event) is dict and "headers" in event and "body" in event:
-        return bento_service_api.handle_aws_lambda_event(event)
+        logger.info('Got prediction request with body "%s"', {event["body"]})
+        prediction = bento_service_api.handle_aws_lambda_event(event)
+
+        # raw json logs
+        logger.debug(event)
```

Comment:
I think it is more common to combine these into one JSON message, e.g.:

```python
import json
logger.info(json.dumps({ 'event': event, 'prediction': prediction["body"], 'status_code': prediction["statudCode"]}))
```

Reply:
great idea, I'll add that